CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+## Test Your Changes
+
+This project's test cases are still WIP. But you can test your changes with a simple Ruby script + a line of command. It will help you test:
```

Comment:
test will be provided soon, so could you remove this line?
Simply "how to try tips" is clear.